### DISCUSSION TITLE
Anthropic Claude 上线啦

### DISCUSSION THREAD

**Discussion by BennyThink** [BurnHair/docs#207] [网站公告]
头顶冒火 小分队上线啦！ https://api.burn.hair/

<img width="1537" alt="iShot_2024-11-10_11 55 11" src="https://github.com/user-attachments/assets/1ce088e7-eba0-4ca0-b370-9c237985e45e">

1. 小分队支持 Anthropic Claude，包括两个分组：官方转发和第三方，可以自行切换

2. 官方分组**价格略贵（1.1-1.2x）**，因为包含了增值税和内容审查接口的成本；第三方接口**价格便宜（0.4x）**，但是不保证任何稳定性

3. 支持 Claude 格式和 OpenAI 兼容格式，接口地址可为 https://burn.hair/v1/messages 或 https://api.burn.hair/v1/messages

4. **系统与主站不互通**，需要重新注册账号，兑换码也不互通

5. 可以自助转移额度，测试阶段仅可转移 $1 

6. 输入会经过 Mistral AI、OpenAI 和 GCP 的文本内容审查，如果命中会返回400系列状态码，因此请不要拿来搞坏事哦

7. 文档可见  https://github.com/BurnHair/docs/blob/master/Claude.md 或首页

<img width="1220" alt="iShot_2024-11-10_18 45 57" src="https://github.com/user-attachments/assets/b9297536-f102-42d5-a960-3391366cf40f">

**Comment by zakizou** [comment]
恭喜，速度真快啊，签到额度挺多,能不能也可以迁移一部分啊

**Reply by BennyThink** [reply]
有说明可以迁移的

**Reply by zakizou** [reply]
![image](https://github.com/user-attachments/assets/bc39dbd5-25f8-498f-926c-334ec0743725)
postman和第三方工具都不能成功调用

**Reply by BennyThink** [reply]
😢你需要把　Text 改成　json

**Reply by zakizou** [reply]
ok,postman可以了
![image](https://github.com/user-attachments/assets/85f44b91-c252-43c8-acb3-942e8a20b5be)
客户端这种好像还都不行

**Reply by BennyThink** [reply]
建议用 Claude格式

**Comment by GrayXu** [comment]
现在支持第三方接口的模型是不是空的呀

**Reply by BennyThink** [reply]
什么意思🤔

**Comment by willhamlam** [comment]
请问一下，遇到以下情况：把令牌分组改成了 vip，还是没有办法调用 vip 分组的模型。

<img width="416" alt="image" src="https://github.com/user-attachments/assets/3c9c959c-6736-4263-b595-4b074e45e542" />

调用的报错：

<img width="736" alt="image" src="https://github.com/user-attachments/assets/c9220a9a-f94c-462d-bc28-e53596142d1b" />

调用 default 的模型是成功的：

<img width="731" alt="image" src="https://github.com/user-attachments/assets/71f39843-12f6-4d84-affd-24d03f377626" />

**Reply by BennyThink** [reply]
已经有 o1 和o3权限了

**Comment by rybtao** [comment]
请问, 我要是想用最新的o1模型, 是不是只能在这个分站才行

**Reply by BennyThink** [reply]
是的 :-)

**Comment by cUp1d** [comment]
Bunny你好，我想问一下现在小分队的额度迁移功能已经取消了吗？首页文档还有，但是充值页面没有看到了

**Reply by BennyThink** [reply]
🐰是的，如有需要可以发邮件，我可以帮忙手动迁移

**Reply by ENIAC1729** [reply]
emm，我好像是github登入的，可以帮助我迁移下吗

**Reply by BennyThink** [reply]
可以的，我需要你提供新站的ID以及你期望转移的金额

**Reply by ENIAC1729** [reply]
id:431 
先转1刀吧，另外咱们有啥群吗，在github上面留言不是太方便

**Reply by BennyThink** [reply]
好了哦。群有的，https://t.me/cat_fight_cat
